lib/providers/autoCompleteHelper.js

```diff
@@ -270,6 +259,10 @@ export default {
 
 					propertyNamesOfType.push(prop.name);
 
+					if (prop.deprecated) {
+						prop.name === prop.name + '|deprecated';
```

Comment:
<!--
  0 failure: 
  1 warning:  lib/providers/aut...
  
  
  DangerID: danger-id-default;
  File: lib/providers/autoCompleteHelper.js;
  Line: 263;
-->

- :warning: lib/providers/autoCompleteHelper.js line 263 – Expected an assignment or function call and instead saw an expression. (no-unused-expressions)